server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -703,6 +703,9 @@ private void buildPods() throws IOException, InterruptedException, ExtenderExcep
                 // We use the same mechanism as when building the extension and create a
                 // manifest context for each pod
                 Map<String, Object> manifestContext = new HashMap<>();
+                manifestContext = ExtenderUtil.mergeContexts(manifestContext, this.platformConfig.context);
+                manifestContext = ExtenderUtil.mergeContexts(manifestContext, this.platformVariantConfig.context);
+                manifestContext = ExtenderUtil.mergeContexts(manifestContext, this.platformAppConfig.context);
```

Comment:
I think we want the platformAppConfig as well?

Reply:
Why not use the already merged `mergedAppContext` from loadManifests() ?
You can make a copy of that and then override your variables below.

Reply:
Good point! Done!